src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -36,6 +36,24 @@ class PBSJobManager extends GridEngineBasedJobManager<PBSCommand> {
         super(executionService, parms)
     }
 
+    @Override
+    boolean checkCompatibilityToHost() {
+        // PBS tests are non trivial. qsub and co are not very helpful, as they don't feature any useful messages.
+        // As we support PBS Torque, we'll check installation folders and for a specific started service.
+        // Checking directories of course can miss, that Torque is installed in a different directory which does
+        // not contain "torque".
+        String cmd = "which qsub;"
```

Comment:
Maybe you should also check something like the version? That may give you much more detailed information.